onnx/defs/schema.cc

```diff
@@ -381,6 +331,27 @@ void OpSchema::Verify(const NodeProto& node) const {
   // Phew. All verifications passed.
 }
 
+void OpSchema::VerifyInputNum(int input_num, const std::string& node_info) const {
+  if (input_num < min_input_ || input_num > max_input_) {
+    fail_check(node_info, " has input size ", input_num, " not in range [min=", min_input_, ", max=", max_input_, "].");
+  }
+
+  if (!num_inputs_allowed_(input_num)) {
+    fail_check(node_info, " has input size ", input_num, " not in allowed input sizes.");
+  }
+}
+
+void OpSchema::VerifyOutputNum(int output_num, const std::string& node_info) const {
```

Comment:
```suggestion
void OpSchema::VerifyOutputNum(int output_num, std::string_view) const {
```

Should we start using `string_view`s?

Reply:
Done.